app/assets/javascripts/autocomplete.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function() {
+
+  $("[data-autocomplete]").each(function() {
+    var input = $(this);
+    var path = input.data("autocomplete");
+
+    input.on('input', function() {
+      if(input.val().length > 0) {
```

Comment:
Agreed, but I couldn't figure out a way to do that with this JQuery plugin. The compromise I found was the `requestDelay` option which will wait for 500ms before sending the request, letting the time to a user to type a certain amount of characters.

This might not be relevant anymore once I switch to React and if I can, I will set a minimum amount of characters before starting the autocomplete.